phpstan.neon

```diff
@@ -1,4 +1,6 @@
 parameters:
   bootstrap: %currentWorkingDirectory%/../../lib/base.php
+  excludes_analyse:
+    - %currentWorkingDirectory%/appinfo/register_command.php
```

Comment:
register_command is non-OOP code. phpstan reports:
```
php -d zend.enable_gc=0 vendor-bin/phpstan/vendor/bin/phpstan analyse --memory-limit=4G --configuration=./phpstan.neon --no-progress --level=5 appinfo lib
 ------ ------------------------------- 
  Line   appinfo/register_command.php   
 ------ ------------------------------- 
  10     Result of && is always false.  
 ------ ------------------------------- 

                                                                                                                        
 [ERROR] Found 1 error                                                                                                  
```
The "if" statement has `isset($application)` but there is no previous set of that var. `phpstan` has no idea that the whole of this PHP file might be included in the middle of something else or... So `phpstan` assumes that the var is never set.